client/src/components/common/Card.svelte

```diff
@@ -12,6 +15,9 @@
 </script>
 
 <article in:slide={{ delay: 300 }} out:slide={{ delay: 300 }}>
+  {#if deletable}
+    <button class="circle deleteButton" on:click={deleteRecipe(id)}><i class="fa-solid fa-trash" /></button>
```

Comment:
card doesn't know it's displaying a recipe so importing something called deleteRecipe that deletes a recipe makes no sense in a card context. The card can emit/dispatch a action on button click and the parent catches the dispatched message and acts on it by deleting a recipe, ingredient or whatever else the card is about